frontend/controller/serviceworkers/sw-primary.js

```diff
@@ -251,9 +251,7 @@ sbp('sbp/selectors/register', {
         })))
         Object.defineProperty(computedGetters, 'currentPaymentPeriodForGroup', {
           get: function () {
-            return (state, getters) => {
-              return (state) => getters.periodStampGivenDateForGroup(state, new Date())
-            }
+            return (state) => getters.periodStampGivenDateForGroup(state, new Date())
```

Comment:
I don't think you can do this. The `getters` object is just a POJO AFAICT.

Calling it like this won't work.

Note `periodStampGivenDateForGroup` takes `state, getters`:

```js
  periodStampGivenDateForGroup (state, getters) {
    return (state, date: string | Date, periods?: string[]): string | void => {
      return periodStampsForDate(date, {
        knownSortedStamps: periods || getters.groupSortedPeriodKeysForGroup(state),
        periodLength: getters.groupSettingsForGroup(state).distributionPeriodLength
      }).current
    }
  },
```